### DISCUSSION TITLE
请问提供微软翻译服务的域名有哪些？

### DISCUSSION THREAD

**Discussion by topkof** [immersive-translate/immersive-translate#1482] [Help]
### 概述 | Summary

近期微软翻译经常失败，加了micrsoft.com代理就正常，但这会增加一下不必要的流量代理，请问提供微软翻译服务的具体域名是什么？

**Comment by theowenyoung** [comment]
edge.microsoft.com

**Reply by topkof** [reply]
实测抓包地址为https://api-edge.cognitive.microsofttranslator.com

**Reply by theowenyoung** [reply]
嗯，有两个，但是似乎这个 microsofttranslator 好像还好？ 这这边测试 edge.microsoft.com 目前偶尔会抽风。

**Comment by topkof** [comment]
> 嗯，有两个，但是似乎这个 microsofttranslator 好像还好？ 这这边测试 edge.microsoft.com 目前偶尔会抽风。

实测很稳定，刚测试的时候可以选择清空DNS缓存并开隐私窗口
